bigtable/google/cloud/bigtable/client.py

```diff
@@ -65,6 +65,13 @@
 READ_ONLY_SCOPE = 'https://www.googleapis.com/auth/bigtable.data.readonly'
 """Scope for reading table data."""
 
+# NOTE: 'grpc.max_message_length' will no longer be recognized in
+# grpcio 1.1 and later.
```

Comment:
I always align on the text, rather than on the note, i.e.

```python
# NOTE: 'grpc.max_message_length' will no longer be recognized in
#       grpcio 1.1 and later.
```

WDYT? Am I crazy?